challenges/prototypes.js

```diff
@@ -1,33 +1,44 @@
 /* ===== Prototype Practice ===== */
 
-// Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. Follow the steps in order to accomplish this challenge.
+// Task: You are to build a cuboid maker that can return values for a cuboid's
+// volume or surface area. Cuboids are similar to cubes but do not have even sides.
+// Follow the steps in order to accomplish this challenge.
 
 /* == Step 1: Base Constructor ==
   Create a constructor function named CuboidMaker that accepts properties for length, width, and height
 */
-
+function CuboidMaker (l, w, h) {
+  this.l = l
+  this.w = w
+  this.h = h
+}
 
 /* == Step 2: Volume Method ==
   Create a method using CuboidMaker's prototype that returns the volume of a given cuboid's length, width, and height
-  
+
   Formula for cuboid volume: length * width * height
 */
-
+CuboidMaker.prototype.volume = function(l, w, h) {
```

Comment:
The function needs zero parameters, as this is a method on the CuboidMaker, is has access to the properties via the `this` keyword.